packages/d3fc-axis/src/axisTickUtils.js

```diff
@@ -0,0 +1,25 @@
+// these utilities capture some of the relatively complex logic within d3-axis which 
+// determines the ticks and tick formatter based on various axis and scale
+// properties: https://github.com/d3/d3-axis#axis_ticks 
+
+const identity = d => d;
+
+const tryApply = (scale, fn, args, defaultVal) =>
+    scale[fn] ? scale[fn].apply(scale, args) : defaultVal;
+
+const ticksArrayForAxis = axis =>
+    axis.tickValues() == null
+        ? tryApply(
```

Comment:
Perfect use case for https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator if you fancy doing the build dance?

Reply:
build dance complete 👍